extras/rya.export/export.accumulo/src/.gitignore

```diff
@@ -0,0 +1 @@
+gen/
```

Comment:
Can you add this to the root .gitignore?  We have an issue with another project too.